api/src/main/resources/application.properties

```diff
@@ -12,12 +12,12 @@
 # Note: we use a custom property name rather than spring.datasource as we're not
```

Comment:
Is this comment still accurate? Why the switch in prefix?

Reply:
Good catch - this is no longer necessary. Updated docs here and in public-api.